### PR TITLE
Basic Code

### PR DESCRIPTION
**Pull request by jvejar1**
# Summary
Here is the code that handles the basic functionalities of the program, along with an ubiquitous language
# Details
- Basic functions that loads any file model and stores it on memory
- A function that delivers ore statistics for the loaded model
- A terminal interface that allows the user to interact with the software

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -43,12 +43,71 @@ def load_file_menu():
...
-    return file_name, coordinates, weight, grades
+        units[grade_name] = input ('Enter the units for the grade (ppm or %):')
+    return file_name, coordinates, weight, grades, units
 
 
+def show_block_info(block,units):
+    print('')
+    print ('Block info: ')
+    print ('Weight\t:',block['weight'])
+    if (block['ores']):
```
Comment (vreyn) [inline main.py]: Altough `ore` is a valid mining concept, for the Ubiquitous language of your project we defined `mineral` as the concept to use. Your code should reflect this and use only `mineral` and not `ore` (currently you are using both)

**Comment by sirjla** [comment]
[Menu for loading the file](https://trello.com/c/fYgpk5Ut/2-menu-for-loading-the-file)

**Comment by sirjla** [comment]
[Menu for displaying block information](https://trello.com/c/QC3v3IQB/3-menu-for-displaying-block-information)

**Comment by sirjla** [comment]
[Display the stats in console](https://trello.com/c/JWRhaSCI/5-display-the-stats-in-console)

File: main.py [hunk 2]
```diff
@@ -76,9 +76,9 @@ def show_stats(blocks,units):
                 total_weight += block_weight
                 blocks_number += 1
 
-                for ore in block['ores'].keys():
-                    ore_metric = float(block['ores'][ore])
-                    ore_unit = units[ore]
+                for mineral in block['minerals'].keys():
+                    ore_metric = float(block['minerals'][mineral])
```
Comment (vreyn) [inline main.py]: the are some "ores" left in this block

**Comment by vreyn** [comment]
You should add a description to the PR, and proabably rename it to reflect the additional code added

**Pull request merged**